code-samples/calendar-event/node/update-calendar-event/index.js

```diff
@@ -0,0 +1,34 @@
+const dotenv = require("dotenv");
+const Nylas = require("nylas");
+
+dotenv.config();
+
+// configuring the Nylas client
+Nylas.config({
```

Comment:
@destryptor we need to update the code sample to reflect the Nylas Node SDK.

So here is what the [v2 SDK supports](https://github.com/nylas/nylas-nodejs/blob/cb42d16b3c597d4297e71303066f2266a66084ee/src/config.ts#L18) for configuration. This is what you have now.

We need to use the Nylas config for the latest SDK, v3, [shown here.]
(https://github.com/nylas/nylas-nodejs/blob/v7.0.0-beta/src/config.ts#L7)

This PR was not tested against the v3 Dashboard, so we need to make updates and test before it can be approve.d